spark/v3.3/build.gradle

```diff
@@ -27,6 +27,16 @@ def sparkProjects = [
     project(":iceberg-spark:iceberg-spark-runtime-${sparkMajorVersion}_${scalaVersion}"),
 ]
 
+configure(sparkProjects) {
+  configurations {
+    all {
+      resolutionStrategy {
+        force 'com.fasterxml.jackson.core:jackson-databind:2.13.4.2'
+      }
+    }
+  }
+}
```

Comment:
To fix:
```
```
> Could not resolve all files for configuration ':iceberg-spark:iceberg-spark-3.3_2.12:compileClasspath'.

   > Could not find com.fasterxml.jackson:jackson-bom:2.13.4.1.
     Searched in the following locations:
       - https://repository.apache.org/content/repositories/orgapachespark-1430/com/fasterxml/jackson/jackson-bom/2.13.4.1/jackson-bom-2.13.4.1.pom
       - https://repo.maven.apache.org/maven2/com/fasterxml/jackson/jackson-bom/2.13.4.1/jackson-bom-2.13.4.1.pom
       - file:/home/runner/.m2/repository/com/fasterxml/jackson/jackson-bom/2.13.4.1/jackson-bom-2.13.4.1.pom
     Required by:
         project :iceberg-spark:iceberg-spark-3.3_2.12 > org.apache.avro:avro:1.11.1 > com.fasterxml.jackson.core:jackson-databind:2.13.4.1
   > Could not find com.fasterxml.jackson:jackson-bom:2.13.4.1.
     Searched in the following locations:
       - https://repository.apache.org/content/repositories/orgapachespark-1430/com/fasterxml/jackson/jackson-bom/2.13.4.1/jackson-bom-2.13.4.1.pom
       - https://repo.maven.apache.org/maven2/com/fasterxml/jackson/jackson-bom/2.13.4.1/jackson-bom-2.13.4.1.pom
       - file:/home/runner/.m2/repository/com/fasterxml/jackson/jackson-bom/2.13.4.1/jackson-bom-2.13.4.1.pom
     Required by:
         project :iceberg-spark:iceberg-spark-3.3_2.12 > org.apache.arrow:arrow-vector:7.0.0 > com.fasterxml.jackson.core:jackson-annotations:2.13.4
         project :iceberg-spark:iceberg-spark-3.3_2.12 > org.apache.avro:avro:1.11.1 > com.fasterxml.jackson.core:jackson-core:2.13.4
Deprecated Gradle features were used in this build, making it incompatible with Gradle 8.0.
```
```

Reply:
Thanks for the fix. The issue is that jackson-databind 2.13.4.1 depends on the non-existed lib, jackson-bom:2.13.4.1. I guess jackson team fixed that with jackson-databind 2.13.4.2